### PR TITLE
Document group state changes when in cases with one specific state

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Document group state changes when in cases with one specific state

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115866
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/group.markdown [hunk 1]
```diff
@@ -405,7 +405,9 @@ When member entities all have a single `on` and `off` state, the group state wil
 | person         | home     | not_home |
 | media_player   | ok       | problem  |
 
-When a group contains entities from domains that have multiple `on` states or only use `on` and `off`, the group state will be `on` or `off`.
+When a group contains entities from domains that have multiple `on` states, and more than one of these `on` states are actively used, or only use `on` and `off`, the group state will be `on` or `off`.
+
+If for example a group has multiple `vacuum` entities, that are all `cleaning`, the group state `on` state will be `cleaning`, of if multiple `lock` entities are `unlocked`, the group `on` state will be `unlocked`.
```
Comment (c0ffeeca7) [inline source/_integrations/group.markdown]:
```suggestion
If for example a group has multiple `vacuum` entities, that are all `cleaning`, the group state `on` state will be `cleaning`, or if multiple `lock` entities are `unlocked`, the group `on` state will be `unlocked`.
```

File: source/_integrations/group.markdown [hunk 2]
```diff
@@ -405,7 +405,9 @@ When member entities all have a single `on` and `off` state, the group state wil
 | person         | home     | not_home |
 | media_player   | ok       | problem  |
 
-When a group contains entities from domains that have multiple `on` states or only use `on` and `off`, the group state will be `on` or `off`.
+When a group contains entities from domains that have multiple `on` states, and more than one of these `on` states are actively used, or only use `on` and `off`, the group state will be `on` or `off`.
```
Comment (c0ffeeca7) [inline source/_integrations/group.markdown]:
```suggestion
When a group contains entities from domains that have multiple `on` states, the **group** state will be `on` or `off` if one of the following is true:
- More than one of these `on` states are actively used.
- The entities only use `on` and `off`.
```
This sentence is a bit complex. Trying to break this up into a list in an attempt to make it easier to read. If it is incorrect or you don't like it for some other reason, feel free to ignore or change.

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/group.markdown (hunk 2 above)
Comment (jbouwh) [reply source/_integrations/group.markdown]: Thnx

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @jbouwh :+1:

**Comment by jbouwh** [comment]
Thnx!

**Pull request merged**